src/m1_facilitator/models/Anchor.ts

```diff
@@ -79,11 +76,7 @@ export default class Anchor extends Comparable<Anchor> {
    *          `< 0` if the current object is less than the given one.
    */
   public compareTo(other: Anchor): number {
-    if (this.anchorGA.localeCompare(other.anchorGA, 'en', { sensitivity: 'base' }) !== 0) {
-      return this.anchorGA.localeCompare(other.anchorGA, 'en', { sensitivity: 'base' });
-    }
-
-    return this.lastAnchoredBlockNumber.comparedTo(other.lastAnchoredBlockNumber);
+    return this.anchorGA.localeCompare(other.anchorGA, 'en', { sensitivity: 'base' });
```

Comment:
Do you see any concern with case sensitive comparison?

Reply:
Ethereum addresses are represented in HEX and hence they are not case sensitive. Or I've missed your point?

Reply:
Checking the address as case insensitive is correct in this context.